modules/game/addgame.rb

```diff
@@ -0,0 +1,43 @@
+module AddGame
+  def add_game
+    publish_date = input_date
+    multiplayer = input_multiplayer
+    last_played_at = input_last_played_at
+    game = Game.new(publish_date, multiplayer, last_played_at)
+    @games << game
+    author = add_author
+    game.add_author(author)
```

Comment:
- Good job done so far, after implementing the custom setter method, to avoid errors in your code, please change this line to:

```ruby
   game.author = author
```